### PR TITLE
Applied JSON  in search Bike  feature & Login, Logout feature

### PR DESCRIPTION
**Pull request by Dilshad21**
Kindly review my overall code and give suggestion in order to improve my code.

### CODE REVIEW

File: src/main/java/in/dilshad/servlet/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	public LoginServlet() {
+		super();
+	}
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse
+	 *      response)
+	 */
+	@Override
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/dilshad/servlet/LoginServlet.java:33]: doPost

File: src/main/java/in/dilshad/servlet/SearchByPlateNoJsonServlet.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		Logger.println("Approach #2: GSON JAR \n" + json);
+
+		// Step 3: Write the json in response and flush it
+		try {
+			PrintWriter out = response.getWriter();
+			out.print(json);
+			out.flush();
+		} catch (IOException e) {
+			// Sonar cloud tells to put try catch block
+		}
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/dilshad/servlet/SearchByPlateNoJsonServlet.java:59]: write e.getMessage()

File: src/main/webapp/displayByPlateNoJSON.jsp [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		
+		let content = "";
+		
+			content += "<tr><td>" + bike.bikeManufacturer + "</td><td>"+ bike.bikeModel + "</td><td>" + bike.bikeColor + "</td><td>" + bike.bikePrice +"</td><td>" + bike.km +"</td><td>" + bike.manufactureYear + "</td></tr>";
+			console.log(bike.bikeManufacturer);
+			console.log(bike.bikeModel);
+			console.log(bike.bikeColor);
+			console.log(bike.bikePrice);
+			console.log(bike.km);
+			console.log(bike.manufactureYear);
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/displayByPlateNoJSON.jsp:59]: console.log(bike);

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**